### PR TITLE
updated step-by-step instructions for meta.yml

### PR DESCRIPTION
**Pull request by eqfinney**
Updated the step-by-step instructions in the recipes.rst to include information about source code format and to point the new user toward more detailed sections of the documentation as relevant.

### CODE REVIEW

File: src/recipe.rst [hunk 1]
```diff
@@ -42,11 +47,12 @@ Step-by-step Instructions
    example/meta.yaml>`_
    file from the example directory. Please leave the example directory
    unchanged!
-4. Edit the copied recipe (meta.yml) as needed.
-5. Generate the SHA256 key as described in the example recipe using the
-   ``openssl`` tool. As an alternative you can also go to the package
-   description on `PyPi <https://pypi.org>`_ from which you can directly copy
-   the SHA256.
+4. Edit the copied recipe (meta.yml) as needed. For details, see 
+   `Writing the ``meta.yml`` <https://conda-forge.org/docs/meta.html>`
```
Comment (msarahan) [inline src/recipe.rst]: I think these inner backticks may throw off the syntax.  Try quotes instead?

Comment (eqfinney) [reply src/recipe.rst]: Good call, fixing that.

**Comment by msarahan** [comment]
Good improvements, thanks for working on this.

**Comment by ocefpaf** [comment]
Thanks @eqfinney! Our docs really needed that!!

**Pull request merged**